### PR TITLE
AP-226 Other Assets page for providers

### PR DESCRIPTION
**Pull request by stephenrichards**
The means test pages for passported applicants are filled in by the provider (as opposed to the 
applicant themselves.  This PR adds the OtherAssetsController in the provider namespace along 
with the appropriate vies (rendering partials from the corresponding page in the Citizens namespace)

[Link to story]https://dsdmoj.atlassian.net/browse/AP-226

Added the controller, and show page, and moved the form in the citizens page to a partial so that it 
can be shared by the Provider and Citizens other assets controllers.

## Checklist

Before you ask people to review this PR:

- [x] Tests and rubocop should be passing: `bundle exec rake`
- [x] Github should not be reporting conflicts; you should have recently run `git rebase master`.
- [x] There should be no unneccessary whitespace changes. These make diffs harder to read and conflicts more likely. 
- [x] The PR description should say what you changed and why, with a link to the JIRA story.
- [x] You should have looked at the diff against master and ensured that nothing unexpected is included in your changes.
- [x] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: app/controllers/providers/other_assets_controller.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      if @form.save
+        render plain: 'Navigate to next question after any other assets'
+      else
+        render :show
+      end
+    end
+
+    private
+
+    def legal_aid_application
```
Comment (tmckernan) [inline app/controllers/providers/other_assets_controller.rb]: You don't need this method if you add `include Providers::ApplicationDependable`

File: spec/requests/providers/other_assets_spec.rb [hunk 2]
```diff
@@ -0,0 +1,120 @@
+require 'rails_helper'
+
+RSpec.describe 'provider other assets requests', type: :request do
+  let(:application) { create :application, :with_applicant }
+  let(:oad) { application.create_other_assets_declaration! }
+  let(:application_id) { application.id }
+
+  before { get providers_legal_aid_application_other_assets_path(application) }
+
+  describe 'GET citizens/own_home' do
```
Comment (tmckernan) [inline spec/requests/providers/other_assets_spec.rb]: Change the describe  from Citizen to Provider i.e  `Get Provider providers/applications/:legal_aid_application_id/own_home`

File: spec/requests/providers/other_assets_spec.rb [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    it 'returns http success' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'displays the show page' do
+      expect(response.body).to include I18n.t('providers.other_assets.show.h1-heading')
+    end
+  end
+
+  describe 'PATCH citizens/own_home' do
```
Comment (tmckernan) [inline spec/requests/providers/other_assets_spec.rb]: Change the describe from Citizen to Provider

File: app/controllers/providers/other_assets_controller.rb (hunk 1 above)
Comment (stephenrichards) [reply app/controllers/providers/other_assets_controller.rb]: Done

**Pull request merged**